panels/config/ha-panel-config.html

```diff
@@ -49,6 +50,14 @@
           hass='[[hass]]'
         ></ha-config-section-core>
 
+        <template is='dom-if' if='[[computeIsThemesLoaded(hass)]]'>
```

Comment:
If themes list is empty - `Object.keys(hass.themes.themes).length` no point in showing this UI